src/App.js

```diff
@@ -2,12 +2,18 @@ import "./App.css";
 import React from "react";
 import { Navbar } from "./Components/Navbar/Navbar";
 import HomePage from "./Pages/HomePage/HomePage";
-
+import { Routes, Route } from "react-router-dom";
+import { Product } from "./Pages/Product/Product";
+import Mockman from "mockman-js";
 function App() {
   return (
     <div className="App">
       <Navbar />
```

Comment:
You can define navbar in the home page only and can use app. js for routes only

Reply:
But then I need to import and use the navbar on every page separately. So to avoid that I have done that.